### ISSUE TITLE
Trans values and Plurals broken in production build

### ISSUE THREAD

**Issue by natmegs** [lingui/js-lingui#386]
**Describe the bug**
When serving a production build, usages of `Trans` macro with values, as well as `Plural`s, are rendered as they appear in messages catalog instead of with appropriate replacements. It appears to be a problem with the production bundling of the `@lingui/core` package.

**To Reproduce**
Minimal set up with code sample below. Run `npm i` before starting.

To see it working in development:
- run `npm start`
- You should see 
```
Some string with variable
Only one
```

To see if broken with production build:
- run `npm run build`
- run `serve -s build`
- You should see 
```
Some string with {variable}
{0, plural, one {Only one} other {More than one}}
```

To fix with production build:
- Find `@lingui/core/index.js` in `node_modules`
- Change file to serve development bundle always
```
if (process.env.NODE_ENV === "production") {
  module.exports = require("./cjs/core.development.js")
} else {
  module.exports = require("./cjs/core.development.js")
}
```
- run `npm run build`
- run `serve -s build`
- You should see 
```
Some string with variable
Only one
```

Minimal set up:

```
import React, { Component } from 'react';
import { Trans, Plural } from "@lingui/macro";
import { I18nProvider } from "@lingui/react";
import logo from './logo.svg';
import './App.css';
import catalogEn from "./locales/en/messages.js";

class App extends Component {
  render() {
    const variable = "variable";
    return (
      <div className="App">
        <I18nProvider language="en" catalogs={{ cs: catalogEn }}>
          <header className="App-header">
            <img src={logo} className="App-logo" alt="logo" />
            <Trans>
              Some string with {variable}
            </Trans>
            <p>
              <Plural 
                value={1}
                one={`Only one`}
                other={`More than one`}
              />
            </p>
          </header>
        </I18nProvider>
      </div>
    );
  }
}

export default App;
```

Find minimal reproduction repo [here](https://github.com/natmegs/lingui-repro) 

**Expected behavior**
Expect `Trans` values to be replaced and `Plural` to be formatted correctly.

**Additional context**
Minimal reproduction created using CRA. Problem originally noticed on non-minimal repo that doesn't use CRA.

**Comment by tricoder42** [comment]
Hey @natmegs, I wonder if it's similar issue as https://github.com/lingui/js-lingui/issues/385#issuecomment-436263590? Do you have all messages translated in the language you're trying to use?

**Comment by natmegs** [comment]
Hi @tricoder42, seems similar but all messages are extracted to the message catalog. When I change the `@lingui/core/index.js` file in node_modules to use the development bundle instead of production bundle it works as expected.

**Comment by tricoder42** [comment]
I see. Do you load compiled message catalogs to your app? That might be another issue, in development messages are compiled on-the-fly

**Comment by natmegs** [comment]
Yes I'm using the catalog from `/locales/en/messages.js` and ran lingui compile before building.

**Comment by tricoder42** [comment]
Ah, yeah sorry, I see it now in the example.

This line looks suspicious:
```
<I18nProvider language="en" catalogs={{ cs: catalogEn }}>
```

There should be `en: catalogEn` to load catalog for `en` locale:

```
<I18nProvider language="en" catalogs={{ en: catalogEn }}>
                                        ^-- en here
```

**Comment by natmegs** [comment]
Ah yes you're right that solved it for the reproduction case but I've been trying to figure out why my original case was still broken. Turned out to be because of issue described in #382 because I didn't import `Trans`. Thanks for the help!

**Comment by mbrevda** [comment]
> There should be en: catalogEn to load catalog for en locale
Even if en is the default locale?
